.gitignore

```diff
@@ -50,3 +50,6 @@ next-env.d.ts
 /src/lib/contracts/generated
 
 dist
+
+# version manager
+.tool-versions
```

Comment:
this is a tool I use to manage my node versions. If you change the version locally it adds a .tool-versions file and I didn't want to commit it